NineChronicles.Headless/GraphTypes/IssueTokenItemsInputType.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace NineChronicles.Headless.GraphTypes.Input
 {
-    public class ItemIdAndCountInputType : InputObjectGraphType<(int itemId, int count)>
+    public class IssueTokenItemsInputType : InputObjectGraphType<(int itemId, int count, bool tradable)>
     {
-        public ItemIdAndCountInputType()
+        public IssueTokenItemsInputType()
         {
             Name = "ItemIdAndCountInput";
```

Comment:
It should update this line too.

Reply:
Resolved by [6337e6a](https://github.com/planetarium/NineChronicles.Headless/pull/2530/commits/6337e6ad64763e5960d976217e765bf79faba815)